File: challenges/2-idade-em-dias/python/virusxdrj/idade_em_dias.py
from sys import argv


def days_to_years(days: int) -> None:

    years: int = days // 365
    days: int = days % 365

    months: int = days // 30
    days: int = days % 30

    print(f"{years} ano(s)\n{months} mes(es)\n{days} dia(s)")


if __name__ == "__main__":
    days_to_years(int(argv[1]))

File: challenges/2-idade-em-dias/javascript/pliniocode/idade-em-dias.js
const inputDays = process.argv[2]

let years = 0;
let months = 0;
let days = 0;

if (inputDays >= 365) {
    years = Math.floor((inputDays / 365));
}

const restDaysToMonth = inputDays % 365;

if (restDaysToMonth >= 30) {
    months = Math.floor((restDaysToMonth / 30))
}

days = (restDaysToMonth % 365) % 30

console.log(`${years} ano(s)`); 
console.log(`${months} mes(es)`); 
console.log(`${days} dia(s)`); 
